File: Arrays & Hashing/Two Sum  .py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        a=1
        for i in range (len(nums)):
            for c in range(a,len(nums)):
                if (nums[i]+nums[c])==target:
                    return [i,c]
            a=a+1

File: Arrays & Hashing/c++/Two Sum  .cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        for (int i = 0; i < nums.size(); i++) {
            int n = target - nums[i];
            auto it = find(nums.begin() + i + 1, nums.end(), n);
            if (it != nums.end()) {
                return {i, static_cast<int>(distance(nums.begin(), it))};
            }
        }
        return {};
    }
};
